src/shared/components/palette/history/event-log/event-log-details.component.spec.ts

```diff
@@ -108,13 +108,12 @@ describe('EventLogDetails', () => {
     fixture.detectChanges();
   }));
 
-  xit('should render a table with the case details', () => {
+  it('should render a table with the case details', () => {
     let rows = de.queryAll($TABLE_ROWS);
```

Comment:
@sakanam - although not part of your change, please change this to a `const` instead.